composer.json

```diff
@@ -41,7 +41,7 @@
     }
   ],
   "require": {
-    "basho/riak": "3.1.*",
+    "basho/riak": "dev-develop",
```

Comment:
@christophermancini I'm assuming this will stay on `dev-develop` until `riak-php-client` and this lib are released.